### ISSUE TITLE
[Active] D0

### ISSUE THREAD

**Issue by Floofball** [ProbablyTez/Agony#374]
"Reroll reality"

Item Pool: Secret and Error Rooms (Rare special item, like Godhead, Sacred Heart, etc.)

Single use item which, once used, resets your run to the first floor, keeping your items but resetting all rooms, floors, bosses, and Angel/Devil precedents. Also applies the Chaos effect.

Additionally, makes the Void floor much more likely to spawn than normal from killing bosses, with the downside that you will ALWAYS be sucked into the portal.

Basically a souped up version of the R Key from AB-.

**Comment by dedChar** [comment]
Greed/Donation machines are disabled when mods are enabled so no need to unstuck them. 
This item would have to be really really rare, or else everyone will just go: Get as many items as possible > Find D0 > Win

**Comment by Floofball** [comment]
@dedChar Hmm. In that case, I'll remove it from the item pool and just keep it as a rare Secret/Error Room item (special item, specifically).
